taichi/runtime/llvm/runtime.cpp

```diff
@@ -1556,7 +1556,7 @@ void stack_push(Ptr stack, size_t max_num_elements, std::size_t element_size) {
 // TODO: make here less repetitious.
 #define DEFINE_SET_PARTIAL_BITS(N)                                            \
   void set_partial_bits_b##N(u##N *ptr, u32 offset, u32 bits, u##N value) {   \
-    u##N mask = ((((u##N)1 << bits) - 1) << offset);                          \
+    u##N mask = ((~(u##N)0) << (N - bits)) >> (N - offset - bits);            \
```

Comment:
Maybe add some comment here like the comment in this PR or add a link to this PR here?